sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Added ability to create a composed model from the `FormTrainingClient` by calling method `StartCreateComposedModel`.
 - Added the properties `ModelName` and `Properties` to types `CustomFormModel` and `CustomFormModelInfo`.
 - Added property `ModelName` to `TrainingOptions` and new type `CreateComposedModelOptions`.
+- Added support for pre-built business card recognition.
```

Comment:
nit: is there any way to make this one stand out more, it seems more important than adding properties. Maybe include it at the top of the list?

Reply:
Good point. 
Updated